### DISCUSSION TITLE
Error saat Run App

### DISCUSSION THREAD

**Discussion by ageljonathan** [teamalgoritma/community#420] [Q&A] [answered]
Waktu saya run app muncul error seperti ini

Error in registerShinyDebugHook(params) : 
  attempt to set an attribute on NULL

Kira2 kenapa ya
![Capture](https://user-images.githubusercontent.com/112919414/198505721-5f83df29-faaa-43c8-920c-7c842894fbd5.JPG)

**Comment by ranydwi** [comment, marked as answer]
Dear Pak @ageljonathan 

Merujuk pada sumber yang saya dapatkan disini https://github.com/HenrikBengtsson/future/issues/158, hal ini disebabkan karena data maupun code yang bapak gunakan melebihi batas yang shiny dapat gunakan yaitu 5mb. Apabila data Bapak memang sudah cukup besar, silahkan set size `shiny.maxRequestSize ` menggunakan code berikut `options(shiny.maxRequestSize=200*1024^2)` pada bagian atas file ui.R maupun server.R Bapak.

Semoga membantu.
Team Algoritma
